CMakeLists.txt

```diff
@@ -39,19 +39,20 @@ endif()
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CPP_STANDARD_FLAGS}")
 
 configure_file(cmake/fccrun.in ${CMAKE_BINARY_DIR}/fccrun)
-configure_file(cmake/this_fccsw.sh.in ${CMAKE_BINARY_DIR}/this_fccsw.sh)
+configure_file(cmake/setup.sh.in ${CMAKE_BINARY_DIR}/setup.sh)
 
 install(FILES ${CMAKE_BINARY_DIR}/fccrun
         DESTINATION .
         PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ
                     GROUP_EXECUTE GROUP_READ
                     WORLD_EXECUTE WORLD_READ
         RENAME run)
-install(FILES ${CMAKE_BINARY_DIR}/this_fccsw.sh
+
+install(FILES ${CMAKE_BINARY_DIR}/setup.sh
         DESTINATION .
-        PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ
-                    GROUP_EXECUTE GROUP_READ
-                    WORLD_EXECUTE WORLD_READ)
+        PERMISSIONS OWNER_WRITE OWNER_READ
```

Comment:
Don't we want the script to be executable?

Reply:
No, we want the script to be sourced. Executing it means running it in its shell which does not have any effect at all. And having it executable can lead to strange behavior like this: https://github.com/HEP-FCC/FCCSW/issues/355